workloads/amd64/k3s-extra-metadata.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: k3s-etcd-snapshot-extra-metadata
```

Comment:
Can we use one `etcd-snapshot-extra-metadata` file for both the products per arch? They look the same to me

Reply:
true